packages/core/src/components/widgets/AltDateTimeWidget.tsx

```diff
@@ -0,0 +1,18 @@
+import { WidgetProps } from "@rjsf/utils";
+import React from "react";
+import AltDateWidget from "./AltDateWidget";
+
+function AltDateTimeWidget<T = any, F = any>({
```

Comment:
Can you add JSDoc here as well... take a look at how the other, already converted widgets are documented as a guide